### ISSUE TITLE
MegDet: A Large Mini-Batch Object Detector

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#331]
Peng, Chao, Xiao, Tete, Li, Zeming, Jiang, Yuning, Zhang, Xiangyu, Jia, Kai, Yu, Gang, Sun, Jian

The improvements in recent CNN-based object detection works, from R-CNN [11]
and Fast/Faster R-CNN [10, 29] to recent Mask R-CNN [14] and RetinaNet [22],
mainly come from new network, or framework, or loss design. But mini-batch
size, a key factor in the training, has not been well studied. In this paper,
we propose a Large Mini-Batch Object Detector (MegDet) to enable the training
with much larger mini-batch size than before (e.g. from 16 to 256), so that we
can effectively utilize multiple GPUs (up to 128 in our experiments) to
significantly shorten the training time. Technically, we suggest a learning
rate policy and Cross- GPU Batch Normalization, which together allow us to suc-
cessfully train a large mini-batch detector in much less time (e.g., from 33
hours to 4 hours), and achieve even better accuracy. The MegDet is the backbone
of our submission (mmAP 52.5%) to COCO 2017 Challenge, where we won the 1st
place of Detection task.

https://arxiv.org/abs/1711.07240
